README.md

```diff
@@ -30,12 +32,31 @@ Dependencies
 Example Playbook
 ----------------
 
-* You can simply use this role like
+* You can simply use this role like. In this case, the code will upgrade all firmwares.
+```
+- hosts: servers
+  roles:
+     - { role: ansible-role-dell-firmware-upgrade }
+```
+
+* If you plan to do not upgrade all firmwares, then you can use the following code: 
 ```
 - hosts: servers
+  vars:
+    - dell_dsu_update_all_firmware: False
   roles:
      - { role: ansible-role-dell-firmware-upgrade }
 ```
+
+* If you plan to upgrade only iDRAC and BIOS firmwares, then you can use the following code: 
```

Comment:
I would add a link to defaults/main.yml#L34-L52 for the possible components (or just a note about looking them up in that file)

Reply:
I agree. Thanks